.github/workflows/staging-cleanup.yml

```diff
@@ -0,0 +1,37 @@
+name: Staging E2E Cleanup
+
+on:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+env:
+  CI: 1
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  cleanup:
+    name: 'Cleanup E2E Test Files'
+    timeout-minutes: 20
+    runs-on: ubuntu-latest-8-cores-open
+    environment: 'e2e-dotcom'
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Run our setup
+        uses: ./.github/actions/setup
+
+      - name: Cleanup test files
+        run: yarn tsx internal/scripts/dotcom/cleanup-test-files.ts
```

Comment:
We could also call cleanup after we run the test.

Reply:
What about running it at the _start_ of the next test run? That way we keep the test files around in case there's a failure but also we don't need to worry about manually cleaning them up.

Reply:
Yeah, probably best if we go with that.

I'd say let's keep it like it is for now, just so I can run these two workflows separately, then once they are fully tested I'll move it to run at the start of the next test run.